src/clarity-angular/button-group/_button-group.clarity.scss

```diff
@@ -7,24 +7,86 @@
 
 @include exports('button-group.clarity') {
 
-  .btn-group {
+    .btn-group {
+        display: flex;
+        margin-right: $clr_baselineRem_0_5;
 
-    display: flex;
+        .btn {
+            margin: 0;
+        }
 
-    .btn {
-      margin-right: 0;
-    }
+        .btn:not(:first-child) {
+            border-top-left-radius: 0;
+            border-bottom-left-radius: 0;
+        }
 
-    .btn:not(:last-child) {
-      border-top-right-radius: 0;
-      border-bottom-right-radius: 0;
-      border-right: none; /* Prevent double borders */
-    }
+        .btn:not(:last-child) {
+            border-top-right-radius: 0;
+            border-bottom-right-radius: 0;
+        }
 
-    .btn:not(:first-child) {
-      border-top-left-radius: 0;
-      border-bottom-left-radius: 0;
-    }
+        &.btn-primary .btn:not(:last-child),
+        &.btn-success .btn:not(:last-child),
+        &.btn-danger .btn:not(:last-child),
```

Comment:
While I generally approve of all this, I just cleaned up the modal.scss in the Wizard refactor and removed 2+ layers of specificity to the CSS selectors due to code like this.

Please make sure that all this specificity is needed. Specifically the child selectors and nesting.